orchagent/portsorch.cpp

```diff
@@ -345,6 +345,74 @@ bool PortsOrch::setPortMtu(sai_object_id_t id, sai_uint32_t mtu)
     return true;
 }
 
+bool PortsOrch::setPortPvid (Port &port, sai_uint32_t pvid)
+{
+    SWSS_LOG_ENTER();
+    vector<Port> portv;
+
+    if (port.m_type == Port::PHY)
```

Comment:
>   if (port.m_type == Port::PHY) [](start = 1, length = 32)

if the port.m_rid_id is not null, we should return false. since we do not allow set pvid for router interface.

Reply:
done.